website/docs/docs/building-a-dbt-project/building-models/configuring-incremental-models.md

```diff
@@ -196,6 +230,7 @@ select ...
 <Changelog>
 
   - **v0.20.0:** Introduced `merge_update_columns`
+  - **v0.21.0:** Introduced `on_schema_change`
```

Comment:
Could you move this to a new Changelog entry up under `## What if the columns of my incremental model change?`?